CredScanSuppressions.json

```diff
@@ -0,0 +1,94 @@
+{
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
+      "placeholder": "$_AZURE_FLUENT_CREDENTIAL_PLACEHOLDER_$",
+      "_justification": "Credential placeholder used in test session records."
+    },
+    {
+      "file": [
+        "azure\\src\\test\\resources\\myTest.pfx",
+        "azure\\src\\test\\resources\\myTest2.pfx",
+        "azure-mgmt-graph-rbac\\src\\test\\resources\\myTest.pfx",
+        "azure-mgmt-network\\src\\test\\resources\\myTest.pfx",
+        "azure-samples\\src\\main\\resources\\myTest.pfx",
+        "azure-samples\\src\\main\\resources\\myTest2.pfx",
+        "azure-samples\\src\\test\\resources\\webapp2-22a0555911.jsdkdemo-ed879923f.com.pfx",
+        "azure-samples\\src\\test\\resources\\webapp2-47b675097d.jsdkdemo-147119052.com.pfx",
+        "azure-samples\\src\\test\\resources\\webapp2-7807785274.jsdkdemo-e43875813.com.pfx",
+        "azure-samples\\src\\test\\resources\\webapp2-7b760367d6.jsdkdemo-f6b20944c.com.pfx",
+        "azure-samples\\src\\test\\resources\\webapp2-bdb5782344.jsdkdemo-33241795f.com.pfx",
+        "azure-samples\\src\\test\\resources\\webapp2-fd02609431.jsdkdemo-6e121655a.com.pfx"
```

Comment:
A pending PR will change these to fixed name like webapp_managewebappwithdomainssl.prx/cer